src/rtctools/data/timeseries.py

```diff
@@ -0,0 +1,88 @@
+"""Module for processing timeseries data."""
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+import rtctools.data.csv as csv
+import rtctools.data.pi as pi
+import rtctools.data.rtc as rtc
+
+
+def get_timeseries_from_csv(
+    file: Path, delimiter=","
+) -> Tuple[List[datetime], Dict[str, np.ndarray]]:
+    """Read timeseries from csv
+
+    :param file:        Path to CSV.
+    :param delimiter:   CSV column delimiter.
+
+    :returns: A list of time stamps (datetimes) and dict of values.
+    """
+    data = csv.load(file, delimiter=delimiter, with_time=True)
+    times = data[data.dtype.names[0]]
+    values = {key: np.asarray(data[key], dtype=np.float64) for key in data.dtype.names[1:]}
+    return times, values
+
+
+def get_timeseries_from_pi(
+    file: Path, data_config=rtc.DataConfig, is_binary=False, validate=False
+) -> Tuple[List[datetime], Dict[str, np.ndarray]]:
```

Comment:
This method is specific to PI files and so should be in 'PIMixin'

Reply:
I'll move it.